classes/class-bsfrt-readtime.php

```diff
@@ -183,25 +190,31 @@ public function bsf_rt_get_option( $key, $default = false ) {
 	 * Frontend settings.
 	 */
 	public function bsf_rt_init_frontend() {
+		global $post;
+		if ( ! empty($post) && has_shortcode($post->post_content, 'read_meter') ) {
+			$this->bsf_rt_add_default_frontend_css();
+			$this->bsf_rt_add_readtime_styles_content();
+			$this->bsf_rt_styles_loaded_flag = true;
+		}
 
 		if ( false === $this->bsf_rt_check_selected_post_types() ) {
 
 			return;
 		}
-		add_action( 'wp_enqueue_scripts', array( $this, 'bsfrt_frontend_default_css' ) );
+		$this->bsf_rt_add_default_frontend_css();
 		add_filter( 'comments_template', array( $this, 'bsf_rt_marker_for_progressbar' ) );
-
+		
```

Comment:
Remove these unwanted changes from below too

Reply:
Which changes are you referring to here?

Reply:
Empty spaces